modules/dynatrace-activegate/vm.tf

```diff
@@ -24,13 +24,22 @@ data "azurerm_key_vault_secret" "ssh_public_key" {
   key_vault_id = data.azurerm_key_vault.subscription_vault.id
 }
 
+data "azurerm_key_vault_secret" "dynatrace_plugin_storage_key" {
```

Comment:
shouldn't this be a data source on the storage account for the subscription why is it coming from key vault?

Reply:
As the plugins are the same for all envs, it's a single storage account as the source for all installations (and envs). Unless I misunderstand your question that is :-)

Reply:
I think you misunderstand

I mean why do you need to read this from key vault you can just read it as a datasource on the storage account?

Reply:
👍